backend/ticketvise/views/api/statistics.py

```diff
@@ -0,0 +1,127 @@
+from django.db.models import F, Count, Window, OuterRef, Subquery, Avg, ForeignKey, CASCADE
+from django.db.models.functions import TruncDate, TruncMonth, TruncYear, RowNumber, TruncHour, TruncQuarter, TruncWeek, \
+    TruncDay, ExtractHour
+from django.http import JsonResponse
+from rest_framework import serializers
+from rest_framework.generics import get_object_or_404
+from rest_framework.serializers import ModelSerializer
+from rest_framework.views import APIView
+
+from ticketvise.models.comment import Comment
+from ticketvise.models.inbox import Inbox
+from ticketvise.models.label import Label
+from ticketvise.models.ticket import Ticket, TicketStatusEvent, Status
+from ticketvise.models.user import User
+from ticketvise.views.api.security import UserIsInboxStaffMixin
+from ticketvise.views.api.ticket import LabelSerializer
+from ticketvise.views.api.user import UserSerializer
+
+
+class InboxTicketsPerDateTypeStatisticsApiView(UserIsInboxStaffMixin, APIView):
```

Comment:
Alleen een manager heeft permissie om de statistics te zien, is het dan de bedoeling dat een agent wel de api data op kan halen?

Reply:
Gefixt!